controllers/configurationpolicy_controller.go

```diff
@@ -2653,6 +2644,8 @@ func (r *ConfigurationPolicyReconciler) checkAndUpdateResource(
 
 				if !isInform {
```

Comment:
As @dhaiducek pointed out, this only shows the top-level key that didn't match so it's not very useful. So these log messages can be removed entirely if there is a log message that says the update is taking place.